src/plugins/aggregate-od/AggregateOd.vue

```diff
@@ -973,6 +976,24 @@ class MyComponent extends Vue {
     })
   }
 
+  private async processGeojson() {
+    let processFeatures = []
+
+    for (let i = 0; i < this.geojson.features.length; i++) {
+      const data = this.geojson.features[i].properties
```

Comment:
Did you want to leave all these console.log() statements in the code?

Reply:
No. This was a mistake